### PR TITLE
add a blog: Install Kubernetes 1.23, containerd, and Multus CNI the Easy Way

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Signed-off-by: FeynmanZhou <feynman@kubesphere.io>

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
Hi @Felixnoo @24sama @chaunceyjiang,

Could you please help me to review it? This blog is similar to https://kubesphere.io/blogs/install-kubernetes-containerd/, which has good page views and SEO performance. As Kubernetes 1.23 was released this month and it has been supported in KubeKey 2.0.0 Alpha release, I plan to publish it.

File: content/en/blogs/install-kubernetes-containerd-multus.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+createTime: '2021-12-26'
+author: 'Feynman'
+snapshot: '/images/blogs/en/kubekey-containerd/kubernetes-containerd-banner.png'
+---
+
+![k8s-containerd](/images/blogs/en/kubekey-containerd/kubernetes-containerd-banner.png)
+
+[KubeKey](https://github.com/kubesphere/kubekey) is a lightweight and turn-key installer that supports the installation of Kubernetes, KubeSphere and related add-ons. Writtent in Go, KubeKey enables you to set up a Kubernetes cluster within minutes. 
+
+Kubernetes 1.23 [was released on Dec 7](https://kubernetes.io/blog/2021/12/07/kubernetes-1-23-release-announcement/). KubeKey has supported the installation of the latest version Kubernetes in its v2.0.0 alpha release, it also brought some new features such as support Multus CNI, Feature Gates, and easy-to-use air-gapped installation, etc.
```
Comment (Felixnoo) [inline content/en/blogs/install-kubernetes-containerd-multus.md:15]: `release, it also` -> `release, and also`

`support Multus` -> `support for Multus`

File: content/en/blogs/install-kubernetes-containerd-multus.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+Make `kk` executable:
+
+```bash
+chmod +x kk
+```
+
+## Step 3: Enable the Multus CNI installation (Optional)
+
+If you want to custom the installation, for example, enable the Multus CNI installation, you can create an example configuration file with default configurations.
```
Comment (Felixnoo) [inline content/en/blogs/install-kubernetes-containerd-multus.md:149]: `custom` -> `customize`

File: content/en/blogs/install-kubernetes-containerd-multus.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+Otherwise, you can skip this step.
+
+```
+./kk create config --with-kubernetes v1.23.0
+```
+
+A default file `config-sample.yaml` will be created if you do not change the name. Edit the file and here is an example of the configuration file of a Kubernetes cluster with one master node. You need to update the host information and enable Multus CNI. We use a single node for this demo, you can also configure a multi-node Kubernetes cluster as you want, see [Multi-node installation](../../docs/installing-on-linux/introduction/multioverview/) for details.
```
Comment (Felixnoo) [inline content/en/blogs/install-kubernetes-containerd-multus.md:156]: `demo, you` -> `demo, and you`

`want, see` -> `want. See`

**Review by Felixnoo — CHANGES_REQUESTED** [review]
Please see my comments, thanks.

**Comment by FeynmanZhou** [comment]
> Please see my comments, thanks.
@Felixnoo 
Updated, thanks for your review!

**Comment by Felixnoo** [comment]
> > Please see my comments, thanks.
> > @Felixnoo
> > Updated, thanks for your review!

Great. Looks good to me!

**Comment by FeynmanZhou** [comment]
> > > Please see my comments, thanks.
> > > @Felixnoo
> > > Updated, thanks for your review!
> 
> Great. Looks good to me!

@Felixnoo Thanks! Could you pls help to comment `/lgtm`?

File: content/en/blogs/install-kubernetes-containerd-multus.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
+---
+title: 'Install Kubernetes 1.23, containerd, and Multus CNI the Easy Way'
```
Comment (chaunceyjiang) [inline content/en/blogs/install-kubernetes-containerd-multus.md:2]: Maybe "1.23.0" would be better?

Comment (FeynmanZhou) [reply content/en/blogs/install-kubernetes-containerd-multus.md:2]: As I see `1.23` is commonly used in the [Kubernetes official blog](https://kubernetes.io/blog/2021/12/07/kubernetes-1-23-release-announcement/), I think we don't need to emphasize `1.23.0`.

**Pull request merged**